dot/build_spec.go

```diff
@@ -98,10 +99,13 @@ func BuildFromDB(path string) (*BuildSpec, error) {
 		Path:      path,
 		LogLevel:  log.Info,
 		Telemetry: telemetry.NewNoopMailer(),
+		GenesisBABEConfig: &types.BabeConfiguration{
```

Comment:
Will this change depending on chain? For example, is the epoch length the same on polkadot as it is on westend?

Reply:
seems odd to me that the `BabeConfiguration` is in a package called `types`.